line/line/urls.py

```diff
@@ -14,13 +14,14 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from django.conf.urls.static import static
 
 from line import settings
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('users.urls')),
```

Comment:
Нужно выделить иерархию в ростах. Для этого все роуты для пользователя нужно вынести под /user/ в доменном имене
```suggestion
    path('users/', include('users.urls')),
```